crest/Assets/Crest/Crest/Scripts/Shapes/ShapeGerstner.cs

```diff
@@ -222,8 +238,17 @@ static void InitStatics()
 
         void InitData()
         {
+            if (_waveBuffers == null)
             {
                 _waveBuffers = new RenderTexture(_resolution, _resolution, 0, GraphicsFormat.R16G16B16A16_SFloat);
+            }
+            else
+            {
+                _waveBuffers.Release();
+                _waveBuffers.width = _waveBuffers.height = _resolution;
```

Comment:
Not a blocker but why does the resolution get set here?

Reply:
Because it might have changed. I'll move it down as it wouldn't hurt.

Reply:
Done.